sophos_central_cloud/README.md

```diff
@@ -1,42 +1,60 @@
-# Agent Check: Sophos Central Cloud
+# Sophos Central Cloud Integration for Datadog
 
 ## Overview
 
-This check monitors [Sophos Central Cloud][1].
+[Sophos Central][1] is a unified cloud-based management platform provided by Sophos, a company specializing in digital security. It allows users to manage various Sophos security products from a single interface. It's designed to simplify security management for businesses of all sizes by consolidating multiple security products into a single, cloud-based platform.
 
-## Setup
+This integration ingests the following logs:
+
+- Alerts
+- Events
 
-### Installation
+The Sophos Central Cloud integration seamlessly collects all the above listed logs, channeling them into Datadog for analysis. Leveraging the built-in logs pipeline, these logs are parsed and enriched, enabling effortless search and analysis. The integration provides insight into alerts and events through the out-of-the-box dashboards. Additionally, the integration enriches corresponding endpoint details along with alert and event logs through the **get_endpoint_details** flag.
 
-The Sophos Central Cloud check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+## Setup
 
 ### Configuration
 
-1. <List of steps to configure this integration>
+#### Sophos Central Cloud Configuration
+
+1. Login to [**Sophos Central Platform**][2] with your credentials
+2. From Sophos Central Admin, go to **My Products** > **General Settings** > **API Credentials Management**.
+3. Then, click on **Add Credential**.
+4. Provide credential name, select the appropriate role, add an optional description and click on **Add** button. The **API credential Summary** for this credential is displayed.
+5. Click on **Show Client Secret** to show the **Client Secret**.
+6. Copy the **Client ID** and **Client Secret**
+
+#### Sophos Central Cloud DataDog Integration Configuration
 
-### Validation
+Configure the Datadog endpoint to forward Sophos Central Cloud events as logs to Datadog.
 
-<Steps to validate integration is functioning as expected>
+1. Navigate to `Sophos Central Cloud`.
+2. Add your Sophos Central Cloud credentials.
+
+| Sophos Central Cloud Parameters | Description                                                                |
+| ------------------------------- | -------------------------------------------------------------------------- |
+| Client ID                       | The Client ID from Sophos Central Cloud.                                         |
+| Client Secret                   | The Client Secret from Sophos Central Cloud.                                     |
+| Get Endpoint Details            | Keep it "true" to collect endpoint details for Sophos Central Cloud Alert and Event Logs, otherwise "false".                 |
```

Comment:
```suggestion
| Get Endpoint Details            | Set to "true" to collect endpoint details for Sophos Central Cloud Alert and Event Logs, otherwise set to "false".                 |
```

Can you also provide what is the default value (true or false) ?

Reply:
Sure. we added default value details.